menas-web/ui/service/OozieService.js

```diff
@@ -18,19 +18,19 @@ jQuery.sap.require("sap.m.MessageBox");
 
 class OozieDAO {
   static getCoordinatorStatus(sCoordinatorId) {
-    return RestClient.get(`api/oozie/coordinatorStatus/${sCoordinatorId}`);
+    return RestClient.get(`oozie/coordinatorStatus/${sCoordinatorId}`);
```

Comment:
Shouldn't this also be prefixed with apiUrl?

Reply:
It's prefixed [here](https://github.com/AbsaOSS/enceladus/pull/1620/files#diff-0818e6ba5065d9f79bee913ebd671ea33847d6e51943e44385f68e74310fb2b5R51) only once